docs/objetivo-4.md

```diff
@@ -0,0 +1,13 @@
+# Documentación hito 4
+
+## Elección del gestor de tests: pytest
+
+Las posibles opciones eran Robot(el más popular), PyTest, UnitTest y DocTest. Robot a pesar de ser muy completo tenía demasiadas funcionalidades que no se le darán uso en este proyecto y DocTest presenta una sintaxis muy fea que no me acaba de convencer. Entre UnitTest y PyTest elegí PyTest debido a que tiene una mayor popularidad y está muy depurado.
+
+El software elegido para realizar los tests, será la librería PyTest. Tiene una sintaxis muy sencilla (no como DocTest por ejemplo) y se invoca de manera sencilla también, presenta buena documentación por lo que he estado viendo y es muy popular entre desarrolladores de Python, este es un aspecto a tener en cuenta ya que si tenemos dudas que necesitasemos resolver es la comunidad quien nos ayudará.

```

Comment:
No se trata de que me comuniquéis la decisión, sino que mostréis que sabéis elegir una herramienta.